samples/CommunityToolkit.Maui.Sample/CommunityToolkit.Maui.Sample.csproj

```diff
@@ -63,7 +63,7 @@
 
   <PropertyGroup Condition="$(TargetFramework.Contains('-windows'))">
     <OutputType>WinExe</OutputType>
-    <RuntimeIdentifier>win-x64</RuntimeIdentifier>
+    <RuntimeIdentifier>win10-x64</RuntimeIdentifier>
```

Comment:
Oh cool! I haven't seen `win10` as a `RuntimeIdentifier` before. 

Does this change any behavior? Will this still run on Windows 11?

Reply:
https://github.com/dotnet/maui/blob/8a4cf99d92f23c1466ae8e0037c26ac1a4b66d23/src/Templates/src/templates/maui-mobile/MauiApp.1.csproj#L54

Reply:
and yes, it can still run on Windows 11

Reply:
Nice!!